src/mahoji/lib/abstracted_commands/openCommand.ts

```diff
@@ -34,6 +37,17 @@ async function addToOpenablesScores(mahojiUser: MUser, kcBank: Bank) {
 	return new Bank().add(mahojiUser.user.openable_scores as ItemBank);
 }
 
+export function buildClueButtons(loot: Bank | null, perkTier: number) {
+	const components: ButtonBuilder[] = [];
+	if (loot && perkTier > PerkTier.One) {
+		const clueReceived = ClueTiers.filter(tier => loot.amount(tier.scrollID) > 0);
+		if (clueReceived.length > 0) {
+			clueReceived.map(clue => components.push(makeDoClueButton(clue)));
+		}
+	}
+	return components;
+}
```

Comment:
this should probably go in the util file, it could create circular imports and other issues. Generally: we prefer to export things from command files.

Reply:
you can check if theres a circular import with: `yarn build` and `npx madge --circular ./dist`, if any of the files in the list outputted are ones you changed, then probably you made a circular import

Reply:
Moved it to a util file (hopefully a suitable one but let me know if you think it should be moved)
Ran the `npx madge --circular ./dist` and couldn't notice any difference in the return compared to master branch.

Let me know if you feel it should be moved elsewhere though.